src/uni_modules/wot-ui/components/wd-form-item/index.scss

```
@@ -11,8 +11,18 @@ $form-item-error-message-line-height: var(--wot-form-item-error-message-line-hei
 $form-item-error-message-margin-top: var(--wot-form-item-error-message-margin-top, $spacing-super-tight) !default;
 // 占位符文字颜色
 $form-item-placeholder-color: var(--wot-form-item-placeholder-color, $text-placeholder) !default;
+// 禁用文字颜色
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)